da-indexer/da-indexer-server/config/eigenda.toml

```
@@ -33,8 +33,8 @@ polling_interval = 12
 [indexer.da]
 type = "EigenDA"
```

Yeap, that should be the following to work

```suggestion
[indexer.da.eigenda]
```

Also, `[indexer.da.rpc]` on line 41 should be updated to `[indexer.da.eigenda.rpc]`

---

Done in https://github.com/blockscout/blockscout-rs/pull/1307/commits/440622f7b650cda516fd1f766e8662240bfb6640